docs/getting-started/continuous_integration.md

```diff
@@ -14,11 +14,15 @@ fix bugs before they make it into your codebase
 
 ## How it works
 
-CIFuzz works by checking out a repository at the head of a pull request. The
-project's fuzz targets are built and run for
-a definite amount of time (default is 10 minutes). If a bug is found, the
-stack trace as well as the test case are made abailable for download.
- If a crash is not found the test passes with a green check.
+CIFuzz works by checking out a repository at the head of a pull request. For projects
```

Comment:
This is quite verbose and I imagine the majority of people reading this won't read all of it :)

I would suggest breaking this into more manageable chunks, with subsections=

e.g.

```
### Picking which fuzzers to run
...
### Reporting bugs
...